admin-ui/app/redux/sagas/AttributesSaga.js

```diff
@@ -2,24 +2,42 @@ import { call, all, put, fork, takeLatest, select } from 'redux-saga/effects'
 import { isFourZeroOneError, addAdditionalData } from 'Utils/TokenController'
 import {
   getAttributesResponseRoot,
+  setAccessToken,
   toggleInitAttributeLoader
 } from '../features/attributesSlice'
 import { getAPIAccessToken } from 'Redux/features/authSlice'
-import { postUserAction } from 'Redux/api/backend-api'
+import { postUserAction, fetchApiAccessToken } from 'Redux/api/backend-api'
 import {
   FETCH,
 } from '../../audit/UserActionType'
 // } from '../../../../app/audit/UserActionType'
 import AttributeApi from '../api/AttributeApi'
 import { getClient } from 'Redux/api/base'
 import { initAudit } from 'Redux/sagas/SagaUtils'
+import { ATTRIBUTES_TAGS } from 'Utils/PermChecker'
 
 const PERSON_SCHEMA = 'person schema'
 
 const JansConfigApi = require('jans_config_api')
 
 function* newFunction() {
-  const token = yield select((state) => state.authReducer.token.access_token)
+  let token
+  const savedToken = yield select((state) => state.attributesReducerRoot.accessToken)
```

Comment:
`savedToken ` naming is not clear. It can be just `token`. Tak the value directly into the `token` variable if its already present in the redux store.